build.gradle

```diff
@@ -92,7 +92,7 @@ dependencies {
     implementation("org.apache.logging.log4j:log4j-core:${log4jVersion}")
     implementation("org.opensearch.client:opensearch-rest-high-level-client:${opensearchVersion}")
     implementation("org.opensearch.client:opensearch-rest-client:${opensearchVersion}")
-    implementation("org.opensearch.client:opensearch-java:${opensearchVersion}")
+    implementation("org.opensearch.client:opensearch-java:2.1.0-SNAPSHOT")
```

Comment:
For my own knowledge, is there a compatibility matrix for `opensearch-java` vs. `OpenSearch`?

Reply:
Not up to date but it's here: https://github.com/opensearch-project/opensearch-java/blob/main/COMPATIBILITY.md